pkg/printers/internalversion/describe.go

```
@@ -1332,6 +1334,127 @@ func describePersistentVolumeClaim(pvc *api.PersistentVolumeClaim, events *api.E
 	})
 }
 
+type VolumeSnapshotDescriber struct {
+	clientset.Interface
+}
+
+func (d *VolumeSnapshotDescriber) Describe(namespace, name string, describerSettings printers.DescriberSettings) (string, error) {
+	c := d.Storage().VolumeSnapshots(namespace)
+
+	vs, err := c.Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	events, _ := d.Core().Events(namespace).Search(legacyscheme.Scheme, vs)
```

Better not ignore the error here.